tests/ui/abi/bad-custom.rs

```diff
@@ -0,0 +1,73 @@
+//@ check-fail
+//@ only-x86_64
```

Comment:
This probably doesn't need to be restricted since the asm is empty (and the assembler shouldn't see this anyway)

Reply:
I believe we have a `needs-asm` test command or something like it to ensure that the test is only run when `asm!` can be used.

Reply:
yeah I'm using `//@ needs-asm-support`